### ISSUE TITLE
Set higher default values and let users override

### ISSUE THREAD

**Issue by IamRaviTejaG** [curiosity81/brainwallet#2]
I'd suggest to set the default values to a securely high number and let the users override those if they wish.

1. makepriv.sh: Set the default number of times the string is hashes to at least 64 by default.
2. string2addr.sh: Set the default number of times the string is hashed to 64.

Set all the iterations which increase safety to high, and let the user override this number if they wish.

**Comment by curiosity81** [comment]
You are correct. It would be good to have secure default values. In the moment there are none. I will add this in future.
